### DISCUSSION TITLE
what are the other Assets in the Releases besides the msix installer exactly?

### DISCUSSION THREAD

**Discussion by nom4d72** [microsoft/winget-cli#3270] [General]
a simple text file that with random letters that look like something like SHA-1
a compressed folder with .psd1 and .psm1 and other contents
and all the others. you could have at least included a minimal description in the txt file

are they for development? can I use them to Install winget using other methods (knowing that would be true would be awesome because I really need winget right away [read #1793])

I did a search but didn't get any of it. sorry linux user

**Comment by florelis** [comment]
AFAIK:

* 73c12d274a444424aa522445ca75f096_License1.xml
A license file for installing the msixbundle; not sure in what cases you need this
* DesktopAppInstallerPolicies.zip
ADMX policies that let you set Group Policies for winget. Say, if you're the IT admin in an organization and you don't want to let people use winget to install stuff on their machines, you can set a policy for that
* Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle
The package installer. You can install it with double-click, or from the CLI with `Add-AppPackage <path to .msixbundle>`
* Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.txt
SHA 256 of the .msixbundle so you can verify you have the right one
* Source code (zip), Source code (tar.gz)
A snapshot of the repo at the point the release was made

> can I use them to Install winget using other methods (knowing that would be true would be awesome because I really need winget right away

Does the .msixbundle not work for you?

**Reply by florelis** [reply]
I forgot these two:
* Microsoft.WinGet.Client-PSModule.zip
A Powershell module for using winget. There is a [quick start guide for the PS module](https://github.com/microsoft/winget-cli/tree/master/src/PowerShell/Microsoft.WinGet.Client#quick-start-guide), but it has a dependency on the winget package, so it's probably not what you need
* Microsoft.WinGet.Client.txt
SHA256 hash of the .zip file
